spark/src/main/scala/io/spark/ddf/etl/TransformationHandler.scala

```diff
@@ -22,7 +22,12 @@ import io.ddf.etl.{TransformationHandler ⇒ CoreTransformationHandler}
 import io.ddf.exception.DDFException
 import io.spark.ddf.SparkDDF
 
+import java.io.PrintStream
+import com.google.api.client.util.LoggingOutputStream
+
 class TransformationHandler(mDDF: DDF) extends CoreTransformationHandler(mDDF) {
+  // Rsession to use
+  val rsess = Rsession.newInstanceTry(new PrintStream(new LoggingOutputStream(mLog, false), true), null)
```

Comment:
@nhanitvn @Huandao0812 this is where I use LoggingOutputStream that comes from com.google.api-client. I have to redirect Rsession's log to go to mLog.